### PR TITLE
Add json output for unsigned transactions

### PR DESCRIPTION
**Pull request by MrStahlfelge**

### CODE REVIEW

File: lib-api/src/main/java/org/ergoplatform/appkit/UnsignedTransaction.java [hunk 1]
```diff
@@ -37,4 +37,16 @@ public interface UnsignedTransaction extends Transaction {
...
     List<ErgoToken> getTokensToBurn();
+
+    /**
+     * Json representation of this transaction.
+     */
+    @Nonnull
+    String toJson(boolean prettyPrint);
+
+    /**
+     * Json representation of this transaction.
```
Comment (aslesarenko) [inline lib-api/src/main/java/org/ergoplatform/appkit/UnsignedTransaction.java:48]: please add description of parameters

**Review by aslesarenko — CHANGES_REQUESTED** [review]

File: lib-api/src/main/java/org/ergoplatform/appkit/UnsignedTransaction.java (hunk 1 above)
Comment (MrStahlfelge) [reply lib-api/src/main/java/org/ergoplatform/appkit/UnsignedTransaction.java:48]: Are you referring to prettyPrint?

Comment (aslesarenko) [reply lib-api/src/main/java/org/ergoplatform/appkit/UnsignedTransaction.java:48]: yes, all parameters, the meaning is not intuitive

Comment (MrStahlfelge) [reply lib-api/src/main/java/org/ergoplatform/appkit/UnsignedTransaction.java:48]: Tbh I don't understand them, I just followed the implementation for `SignedTransaction`. The parameters were introduced in #73. Maybe @wclaeys can shed some light.

Comment (aslesarenko) [reply lib-api/src/main/java/org/ergoplatform/appkit/UnsignedTransaction.java:52]: hi, surprised you found it problematic :)

**Review by aslesarenko — APPROVED** [review]

**Pull request merged**